static/js/main.js

```diff
@@ -1,7 +1,8 @@
-window.onload = function() {
+var initApplication = (function() {
+    'use strict';
 
     var $menuIcon = document.getElementsByClassName('menu-icon')[0],
-        $offCanva = document.getElementsByClassName('off-canvas')[0];
+        $offCanva = document.getElementsByClassName('off-canvas')[0],
```

Comment:
There's a parsing error here, which I fixed. General practice is to make multiple assignments to prevent accidental parsing issues when grouping declarations.